main.nf

```diff
@@ -18,46 +18,54 @@ def helpMessage() {
 
     The typical command for running the pipeline is as follows:
 
-    nextflow run nf-core/viralrecon --input samplesheet.csv --viral_genome 'NC_045512.2' -profile docker
+    nextflow run nf-core/viralrecon --input samplesheet.csv --host_genome 'human' --viral_genome 'NC_045512.2' -profile docker
 
     Mandatory arguments
       --input [file]                  Comma-separated file containing information about the samples in the experiment (see docs/usage.md)
-      --viral_fasta [file]            Path to Fasta reference for viral genome
       -profile [str]                  Configuration profile to use. Can use multiple (comma separated)
                                       Available: conda, docker, singularity, test, awsbatch, <institute> and more
 
     Generic
       --protocol [str]                Specifies the type of protocol used for sequencing i.e. "metagenomic" or "amplicon". (Default: "metagenomic")
-      --amplicon_fasta                Path to Fasta file containing amplicon sequences
+      --amplicon_fasta [file]         Path to fasta file containing amplicon sequences
 
     References                        If not specified in the configuration file or you wish to overwrite any of the references
-      --host_kraken2_db [file]        Full path to Kraken2 database built from host genome
-      --host_kraken2_name [str]       Name of host genome for building Kraken2 database (Default: "human")
+      --host_genome [str]             Name of genome reference key for human genome
 
-      --viral_genome [str]            Name of iGenomes reference for viral genome
-      --viral_kraken2_db [file]       Full path to Kraken2 database built from viral genome
+      --viral_genome [str]            Name of genome reference key for viral genome
+      --viral_fasta [file]            Path to fasta reference for viral genome
       --viral_gff [file]              Full path to viral gff annotation file
 
       --save_reference [bool]         If generated by the pipeline save the Bowtie2 indices in the results directory (Default: false)
 
+    Kraken2
+      --host_kraken2_db [file]        Full path to Kraken2 database built from host genome
+      --host_kraken2_name [str]       Name of host genome for building Kraken2 database
+
+      --viral_kraken2_db [file]       Full path to Kraken2 database built from viral genome
+      --viral_kraken2_name [str]      Name of viral genome for building Kraken2 database
```

Comment:
Again, a default or example value would be useful here

Reply:
See answer to above comment.